indico/modules/rb_new/schemas.py

```diff
@@ -72,6 +73,12 @@ class Meta:
                   'surface_area', 'latitude', 'longitude', 'telephone', 'key_location', 'max_advance_days',
                   'owner_name', 'available_equipment', 'has_photo', 'verbose_name')
 
+    @post_dump(pass_many=True)
+    def inject_fields(self, data, many):
+        if many:
```

Comment:
Currently this doesn't work after editing a room when just a single room is fetched. You can use this instead so your signal is always called properly:

```
        rooms = data if many else [data]
        signals.rb.rooms_fetched.send(rooms)
```